src/imageio.cs

```diff
@@ -1169,4 +1213,26 @@ interface CGCopyImageSourceOptions {
 		IntPtr kOrientation { get; }
 	}
 #endif
+
+	[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
+	public enum CGImageAuxiliaryDataType {
+		[Field ("kCGImageAuxiliaryDataTypeDepth")]
+		Depth,
+
+		[Field ("kCGImageAuxiliaryDataTypeDisparity")]
+		Disparity,
+	}
+
+	[Partial]
+	[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
+	public interface CGImageAuxiliaryDataInfo {
+		[Field ("kCGImageAuxiliaryDataInfoData")]
+		NSString Data { get; }
+
+		[Field ("kCGImageAuxiliaryDataInfoDataDescription")]
+		NSString DataDescription { get; }
+
+		[Field ("kCGImageAuxiliaryDataInfoMetadata")]
+		NSString kMetadata { get; }
```

Comment:
I named this kMetadata because a Metadata strongly-typed property is already defined for the class in CGImageDestination.cs

```public partial class CGImageAuxiliaryDataInfo {
		public NSData DepthData { get; set; }

		public NSDictionary DepthDataDescription { get; set; }

		public CGImageMetadata Metadata { get; set; }```

I copied how CGImageDestinationOptions was bound for the above.  I can change it to work differently if desired.